dts/riscv/ite/it8xxx2-pinctrl-map.dtsi

```diff
@@ -7,6 +7,44 @@
 #include <dt-bindings/pinctrl/it8xxx2-pinctrl.h>
 
 &pinctrl {
+	/* ADC alternate function */
+	pinctrl_adc0: adc0 {
```

Comment:
note: the `pinctrl_xxx` naming is confusing. The guidelines recommend something like `periph_signal_pin_state` (not all fields may apply), maybe: `adc0_pi0_default`? (not sure about the `pi0` part since I don't know how ITE names pins).

Reply:
Done.